hs_core/models.py

```diff
@@ -270,7 +270,7 @@ def update(cls, element_id, **kwargs):
         if 'description' in kwargs:
             party = cls.objects.get(id=element_id)
             if party.description is not None and kwargs['description'] is not None:
-                if len(party.description) > 0 and len(kwargs['description']) > 0:
+                if len(party.description.strip()) > 0 and len(kwargs['description'].strip()) > 0:
                     if party.description != kwargs['description']:
```

Comment:
@pkdash Right, I understand that, I initially thought if two strings are only differed by white spaces, you probably should regard them as equal rather than different, hence thought it is better to compare to see whether they are equal by stripping out white spaces first. But we may argue the other way also for this case. What do you think? If you think it is OK not to strip out white spaces before comparing, I am OK with that and can give a +1 for this PR.